docs/030_user-guide/015_sdk.md

```diff
@@ -0,0 +1,93 @@
+# Pepr SDK
+
+## `containers`
+
+Returns list of all containers in a pod. Accepts the following parameters:
+
+- **@param peprValidationRequest** The request/pod to get the containers from
+- **@param containerType** The type of container to get
+
+**Usage:**
+
+**_Get all containers_**
+
+```typescript
+let result = containers(peprValidationRequest)
+```
+
+**_Get only the standard containers_**
+
+```typescript
+let result = containers(peprValidationRequest, "containers")
+```
+
+**_Get only the init containers_**
+
+```typescript
+let result = containers(peprValidationRequest, "initContainers")
+```
+
+**_Get only the ephemeral containers_**
+
+```typescript
+let result = containers(peprValidationRequest, "ephemeralContainers")
+```
+
+---
+
+## `getOwnerRefFrom`
+
+Returns the owner reference for a Kubernetes resource. Accepts the following parameters:
+
+- **@param kubernetesResource: GenericKind** The Kubernetes resource to get the owner reference for
+
+**Usage:**
+
+```typescript
+const ownerRef = getOwnerRefFrom(kubernetesResource);
+
```

Comment:
I think you're  missing the backticks to close the typescript block here.  :thinking: